### PR TITLE
CR for #20547 - Add "reload" buttons for redownloading scripts

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6623d0c5527820843d6616f8d971ba2d0bc9a484 available at: https://gist.github.com/af5dc07354b613fe00f1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.dmg

File: libraries/entities/src/EntityItem.h [hunk 1]
```diff
@@ -276,6 +276,10 @@ class EntityItem {
 
     const QString& getScript() const { return _script; }
     void setScript(const QString& value) { _script = value; }
+
+    const quint64 getScriptTimestamp() const { return _scriptTimestamp; }
```
Comment (sethalves) [inline libraries/entities/src/EntityItem.h]: make this:
  quint64 getScriptTimestamp() const { return _scriptTimestamp; }
const means nothing for a return-by-value type.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 93509f4c6e91f475039e90e4ebe2f2d73ae78c6b available at: https://gist.github.com/af5dc07354b613fe00f1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.dmg

**Comment by cozza13** [comment]
I have functionally tested this and it works

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1898c6bd552fa176a853f446eb115323784d5580 available at: https://gist.github.com/af5dc07354b613fe00f1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5154.dmg

File: libraries/entities/src/EntityPropertyFlags.h [hunk 2]
```diff
@@ -55,6 +55,7 @@ enum EntityPropertyList {
     PROP_DAMPING,
     PROP_LIFETIME,
     PROP_SCRIPT,
+    PROP_SCRIPT_TIMESTAMP,
```
Comment (ZappoMan) [inline libraries/entities/src/EntityPropertyFlags.h]: NOTE: this change broke binary compatibility.... we now have corrupted models.svo files

**Pull request merged**